zebra-chain/src/work/difficulty.rs

```diff
@@ -63,9 +63,13 @@ pub struct CompactDifficulty(pub(crate) u32);
 
 impl fmt::Debug for CompactDifficulty {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        // There isn't a standard way to represent alternate formats for the
+        // same value.
```

Comment:
what does this mean? There's a flag for `alternate` format flag on `std::fmt::Formatter` that you can use to setup an alternate format. https://doc.rust-lang.org/stable/std/fmt/struct.Formatter.html#method.alternate

Reply:
We want to show the same data in multiple ways in the Debug format, but the `alternate` format flag allows the calling code to select between different formats.

Reply:
Fixed in 558a9f46b59908e604b00f8b785266c9cbb14410